test/subtype.jl

```diff
@@ -2310,7 +2311,8 @@ end
 
 # try to fool a greedy algorithm that picks X=Int, Y=String here
 @test Tuple{Ref{Union{Int,String}}, Ref{Union{Int,String}}} <: Tuple{Ref{Union{X,Y}}, Ref{X}} where {X,Y}
-@test Tuple{Ref{Union{Int,String,Missing}}, Ref{Union{Int,String}}} <: Tuple{Ref{Union{X,Y}}, Ref{X}} where {X,Y}
+#disabled due to 49857
+#@test Tuple{Ref{Union{Int,String,Missing}}, Ref{Union{Int,String}}} <: Tuple{Ref{Union{X,Y}}, Ref{X}} where {X,Y}
```

Comment:
@N5N3 added this test in #48441. Since this PR changes that code by removing the slow path, I am assuming that it's expected for this test to fail.

The other 2 tests I disabled are for #40865, mentioned above.

Reply:
It's better to mark such things as `skip=true`. `broken=true` would be better, but seeing as this is about a freeze, not freezing seems preferrable.